src/middlewares/authorizeUser.js

```diff
@@ -0,0 +1,17 @@
+import helpers from '../helpers';
+
+const { responseMessage } = helpers;
+/**
+ *
+ * @param {object} roleArray an array of roles
+ * @returns {*} json or next
+ */
+const authorizeUser = roleArray => async (request, response, next) => {
```

Comment:
Nice one

Reply:
Thank you for the commendation, I appreciate your feedback